VisualizationBase/src/overlays/ArrowOverlay.h

```diff
@@ -43,26 +43,22 @@ class VISUALIZATIONBASE_API ArrowOverlay: public Super<Overlay<Item>>
 	public:
 		ArrowOverlay(Item* arrowFrom, Item* arrowTo, const StyleType* style = itemStyles().get());
 		virtual void paint(QPainter* painter, const QStyleOptionGraphicsItem* option, QWidget* widget) override;
+		void setSelected(bool selected);
```

Comment:
`QGraphicsItem` already defines a method called `setSelected` and you are now shadowing that one. Please use a different name, e.g. `setHighlighted`. I guess in that case, it makes sense to change the field name from `selected_` to `highlighted_`.